<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>636</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>ContactsManager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(43, 39, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>511</width>
      <height>621</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTableView" name="tableInventory">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>491</width>
        <height>421</height>
       </rect>
      </property>
      <property name="cornerButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAddNew">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>97</width>
        <height>31</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="text">
       <string>Add New</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBoxTagSearch">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>500</y>
        <width>301</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>TagSearch</string>
      </property>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>39</y>
         <width>71</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>friend</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>108</x>
         <y>39</y>
         <width>61</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>work</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>205</x>
         <y>39</y>
         <width>81</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>so</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_4">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>73</y>
         <width>81</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>school</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_5">
       <property name="geometry">
        <rect>
         <x>108</x>
         <y>73</y>
         <width>81</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>others</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_6">
       <property name="geometry">
        <rect>
         <x>205</x>
         <y>73</y>
         <width>81</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>relatives</string>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="filterEdit">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>20</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="backToPage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>51</x>
        <y>100</y>
        <width>36</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>51</x>
        <y>150</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Surname</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>51</x>
        <y>190</y>
        <width>39</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Phone</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>51</x>
        <y>240</y>
        <width>33</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>51</x>
        <y>280</y>
        <width>36</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Notes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>440</y>
        <width>321</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(252, 10, 25)</string>
      </property>
      <property name="text">
       <string>DELETE</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textEditName1">
      <property name="geometry">
       <rect>
        <x>114</x>
        <y>96</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="textEditSurname1">
      <property name="geometry">
       <rect>
        <x>114</x>
        <y>141</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="textEditPhone1">
      <property name="geometry">
       <rect>
        <x>114</x>
        <y>186</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="textEditEmail1">
      <property name="geometry">
       <rect>
        <x>114</x>
        <y>231</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="textEditNotes1">
      <property name="geometry">
       <rect>
        <x>114</x>
        <y>276</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>79</x>
        <y>310</y>
        <width>361</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Tag</string>
      </property>
      <widget class="QRadioButton" name="radioFriend">
       <property name="geometry">
        <rect>
         <x>31</x>
         <y>50</y>
         <width>101</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>friend</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioWork">
       <property name="geometry">
        <rect>
         <x>141</x>
         <y>50</y>
         <width>101</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>work</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioSchool">
       <property name="geometry">
        <rect>
         <x>31</x>
         <y>82</y>
         <width>101</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>school</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioSO">
       <property name="geometry">
        <rect>
         <x>252</x>
         <y>50</y>
         <width>101</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>so</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioRelative">
       <property name="geometry">
        <rect>
         <x>252</x>
         <y>82</y>
         <width>101</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>relatives</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioOthers">
       <property name="geometry">
        <rect>
         <x>141</x>
         <y>82</y>
         <width>101</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>others</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
